README.rst

```diff
@@ -17,46 +17,49 @@ Pytest container
            :alt: PyPI
            :target: https://pypi.org/project/pytest-container/
 
-A simple `pytest <https://pytest.org>`_ plugin to test Container Images
-via pytest fixtures and `testinfra <https://testinfra.readthedocs.io/en/latest/>`_.
-
-This module provides a set of fixtures and helper functions to ease testing of
-container images leveraging `testinfra
-<https://testinfra.readthedocs.io/en/latest/>`_. Assuming you want to
-automatically spin up a container for a test, then the `container` fixture will
-do exactly that (plus it will cleanup after itself):
+``pytest_container`` is a `pytest <https://pytest.org>`_ plugin
+to test container images via pytest fixtures and `testinfra
+<https://testinfra.readthedocs.io/en/latest/>`_. It takes care of all the boring
+tasks, like spinning up containers, finding free ports and cleaning up after
+tests, and allows you to focus on implementing the actual tests.
+
+The plugin automates the following tasks:
+- automatically pull, launch & stop containers
+- build containers via a :file:`Dockerfile`
+- wait for containers to become healthy before executing tests
+- bind exposed container ports to free ports on the host
+- mount volumes via temporary directories
+- parallel test execution via pytest-xdist
+- dependency between container images
```

Comment:
I feel like there is something missing. dependency resolution, management?

Reply:
Hm, it doesn't really do dependency resolution. All it can do is the following: image B needs A to build, so it will first build A and then B (you can do that recursively as well). But there's no real dependency resolution here (and it will certainly fail if you introduce cycles)